channels/OWNERS

```diff
@@ -1,10 +0,0 @@
-# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
-# This file just uses aliases defined in OWNERS_ALIASES.
-
-approvers:
-  - release-admins
```

Comment:
I'd rather have ART own candidate promotions, and cincinnati-graph-data-approvers only own fast/stable promotions.  But we can't do that until Cincinnati can load channel YAML recursively from `channels/` (which we'd split into `channels/candidate` and `channels/stable` or some such).

Reply:
Sounds like ART is going to push directly to master, so they don't need to get involved in `OWNERS`.  In that case, I'm fine with this PR as it stands.

Reply:
We can come back to it and add ART to the list of approvers when we have ART ready to push to the candidates.